blocks/http/include/gnuradio-4.0/http/all.hpp

```diff
@@ -0,0 +1,6 @@
+#ifndef GNURADIO_BLOCKS_HTTP_ALL_HPP
```

Comment:
Why this header file? Doesn't one usually 'only-include-what-you-need'?!?

Reply:
If we want the blocks to be visible to the registry, they are all needed.

Applications that don't need GRC (or any runtime block instantiation), but only use fixed C++-defined flows are fine to include only the blocks they need. Applications that need to be able to instantiate any known block have to include them all.